.github/workflows/build.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build - JVM
-        run: mvn -B formatter:validate install --file pom.xml
+        run: mvn -B formatter:validate verify --file pom.xml
```

Comment:
@gastaldi This will likely fail in future builds; `install` puts the quarkus-cxf artifacts into the local maven repo, so that they are available for the next step (the native build), and `verify` I believe doesn't do that.  I'm guessing the build passed for now because you set up a cache which coincidentally already had the necessary artifacts.